scripts/ci-entrypoint.sh

```diff
@@ -113,11 +113,6 @@ select_cluster_template() {
         export CLUSTER_TEMPLATE="test/ci/cluster-template-prow.yaml"
     fi
 
-    if [[ -n "${TEST_CCM:-}" ]]; then
```

Comment:
I think you forget to handle code at L53 with `TEST_CCM`. As a result, CCM and CNM related env vars are not set when helm upgrade cloud-controller-manager

Reply:
moved the variables to the top